pages/about.html

```diff
@@ -46,11 +45,11 @@
           
         <div class="desktop-menu">
             <ul class="menu-items">
-            <li><a href="./pages/about.html"  class="mitem">About</a></li>
-            <li><a href="#features"  class="mitem">Features</a></li>
-            <li><a href="#partners"  class="mitem">Partners</a></li>
-            <li><a href="./pages/about.html#email"  class="mitem">Contact Us</a></li>
-            <li><a href="#speaker-container"  class="mitem">Clients</a></li>
+            <li><a href="../pages/about.html"  class="mitem">About</a></li>
```

Comment:
- [ ] i think in the header for the about page in the desktop page you should align the headings and the paragraph to the center so kindly center them as you can see in the original design 💪🏻💪🏻💪🏻

- [ ] Please fix the margin between the elements and make them to have the same margin as the given design 🙏🏻🙏🏻🙏🏻

- Your design
![image](https://user-images.githubusercontent.com/58553711/181495461-aa6a786d-c39c-484a-8396-7a50c30f6315.png)

- Orifinal design
![image](https://user-images.githubusercontent.com/58553711/181495298-20942ee2-5629-43d8-8858-bbb21a45489d.png)